oaknational/Oak-Web-Application
# Feat/lesq 213/render new quiz data

## Description

Adapt OWA component to handle final data format

## Issue(s)

Fixes #

## How to test

1. Go to  https://deploy-preview-1809--oak-web-application.netlify.thenational.academy/beta/teachers-2023/programmes/geography-secondary-ks4/units/test-unit/lessons/lesson-1-in-test-unit

2. Also check the story book under New Question List

## Screenshots


How it should now look:
![image](https://github.com/oaknational/Oak-Web-Application/assets/1030540/bb681a68-b959-43f8-bc59-fe409c1f16e7)


## Checklist

- [ x]  Text questionStem
- [ x]  Image & text questionStem
- [ x]  Text MCQ answers
- [ x]  Image MCQ answers
- [ x]  Short answer
- [x ]  Order
- [ x]  Match
- [x] screen readable
